storybook/.babelrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Looks like the issue on `source/renderer/app/components/wallet/transactions/Transaction.tsx` is related to [babel-plugin-react-intl](formatjs/babel-plugin-react-intl#166). Maybe upgrading to latest version should fix this. 

Also, we could consider removing a few babel plugins in case there are not needed anymore as `preset-typescript` and `preset-env` should cover most of the js spec?